### PR TITLE
Added notes to documentation on ToogleGroup and ToogleButton

### PR DESCRIPTION
**Pull request by Unraveler**
DOCUMENTATION

### Description:

`ToggleButton` has a property called `pressed` that is overridden by the property `selectedName` from `ToogleGroup`. This appears to cause some confusion on some users, so a note to the documentation has been added.

Fixes Issue? #1710 
Examples added? No (not needed)
Tests added? No (not needed)
Docs added? Yes
Would a screenshot be helpful? No

### CODE REVIEW

File: src/Button/ToggleButton/ToggleButton.tsx [hunk 1]
```diff
@@ -26,6 +26,8 @@ interface DefaultProps {
   tooltipProps: AbstractTooltipProps;
   /**
    * The initial pressed state of the ToggleButton
+   * Note: If ToggleButton is set inside a ToggleGroup, this prop will be controlled by selectedName prop on
```
Comment (simonseyock) [inline src/Button/ToggleButton/ToggleButton.tsx:29]:
```suggestion
   * Note: If a ToggleButton is inside a ToggleGroup, this property will be controlled by selectedName property of the
```

File: src/Button/ToggleButton/ToggleButton.tsx [hunk 2]
```diff
@@ -26,6 +26,8 @@ interface DefaultProps {
...
+   * ToggleGroup. Setting in in Toggle button will not work.
```
Comment (simonseyock) [inline src/Button/ToggleButton/ToggleButton.tsx:30]:
```suggestion
   * ToggleGroup and the will property will be ignored.
```

File: src/Button/ToggleGroup/ToggleGroup.tsx [hunk 3]
```diff
@@ -30,6 +30,8 @@ interface BaseProps {
   /**
    * The value fo the `name` attribute of the children to select/press
    * initially.
+   * Note: This prop will have full control over the pressed prop on its children. Setting select/pressed on the
+   * children props directly, will have no effect.
```
Comment (simonseyock) [inline src/Button/ToggleGroup/ToggleGroup.tsx:34]:
```suggestion
   * children props directly will have no effect.
```

Comment (simonseyock) [reply src/Button/ToggleGroup/ToggleGroup.tsx:34]: I am not sure here :) but it should work without `,`

Comment (Unraveler) [reply src/Button/ToggleGroup/ToggleGroup.tsx:34]: It's correct. Better without comma

**Comment by dnlkoch** [comment]
As soon as the lint errors are fixed, this can be merged.

**Pull request merged**